src/WPML_OptionsManager.php

```diff
@@ -547,59 +563,55 @@ private function displayMP3Banner() {
-                <li><?php _e( 'Over 100,000 active users', 'wp-mail-logging' );?></li>
-            </ul>
-            <a
-                class="button button-primary"
-                href="?page=wpml_plugin_log&redirect=free-plan&ref=wml_1"
-                target="_blank"
-            >
-                <?php _e( 'Try MailPoet for free', 'wp-mail-logging' );?>
-            </a>
-            <p>
-                <?php _e( 'Testimonial', 'wp-mail-logging' );?>:
-                <i>
-                    <?php _e( 'Thanks for this awesome plugin, love love love how it integrates with WordPress. I seriously spent days if not weeks on Mailchimp, and still haven’t been able to do what I did on MailPoet in 1 hour!', 'wp-mail-logging' );?>
-                </i>
-                — Kida Shey
-            </p>
-        <?php
+		$banner = '<ul style="list-style-type:disc;list-style-position: inside">';
+		$banner .= sprintf( '<li>%s</li>', __( '50 email templates to choose from', 'wp-mail-logging' ) );
+		$banner .= sprintf( '<li>%s</li>', __( 'Fun email designer', 'wp-mail-logging' ) );
+		$banner .= sprintf( '<li>%s</li>', __( 'Automated email marketing', 'wp-mail-logging' ) );
+		$banner .= sprintf( '<li>%s</li>', __( 'WooCommerce emails', 'wp-mail-logging' ) );
+		$banner .= sprintf( '<li>%s</li>', __( '99.1% success email deliverability', 'wp-mail-logging' ) );
+		$banner .= sprintf( '<li>%s</li>', __( 'Fast and friendly support', 'wp-mail-logging' ) );
+		$banner .= sprintf( '<li>%s</li>', __( 'Over 100,000 active users', 'wp-mail-logging' ) );
+		$banner .= '</ul>';
+		$banner .= sprintf( '<a class="button button-primary" href="%s" target="_blank">%s</a>',
+			'?page=wpml_plugin_log&redirect=free-plan&ref=wml_1',
+			__( 'Try MailPoet for free', 'wp-mail-logging' )
+		);
+		$banner .= sprintf( '<p>%s: <i>%s</i> %s</p>',
+			__( 'Testimonial', 'wp-mail-logging' ),
+			__( 'Thanks for this awesome plugin, love love love how it integrates with WordPress. I seriously spent days if not weeks on Mailchimp, and still haven’t been able to do what I did on MailPoet in 1 hour!', 'wp-mail-logging' ),
+			__( '— Kida Shey', 'wp-mail-logging' )
+		);
+
+		return $banner;
     }
 
     private function displayMP3BannerVersionB() {
-        ?>
-        <p>
-            <?php _e( 'Create beautiful email campaigns and reach your audience in a breeze with the MailPoet  plugin. MailPoet is used by over 300,000 website owners making it the most popular email marketing solution in WordPress. Send beautiful newsletter, notify your readers about last articles and increase your WooCommerce sales!', 'wp-mail-logging' );?>
-        </p>
-        <p>
-            <a href="?page=wpml_plugin_log&redirect=free-plan&ref=wml_2" target="_blank" class="button button-primary">
-                <?php _e( 'Discover MailPoet for free', 'wp-mail-logging' );?>
-            </a>
-        </p>
-        <?php
+		$banner = sprintf( '<p>%s</p>', 
```

Comment:
Could you please explain why this method's approach to output construction needed to change?
If it was not necessary, could you please revert back the original method code?